### ISSUE TITLE
Replace Ace editor with CodeMirror?

### ISSUE THREAD

**Issue by alexwarren** [textadventures/squiffy-editor#3]
As reported [on the forums](http://forum.textadventures.co.uk/viewtopic.php?f=24&t=5319), the Squiffy editor doesn't work with screen readers.

This is entirely a [known issue with the Ace editor](https://github.com/ajaxorg/ace/issues/2164) we're currently using.

Investigate switching to [CodeMirror](https://codemirror.net/) or providing it as an option.

**Comment by alexwarren** [comment]
CodeMirror is no good either - still has accessibility problems.

Best bet is to have an option to replace Ace editor with a standard textarea.

**Comment by alexwarren** [comment]
textarea feature opened as separate issue #7
